wcomponents-theme/src/main/js/wc/file/util.js

```diff
@@ -0,0 +1,126 @@
+define(["wc/dom/uid", "wc/file/getMimeType", "wc/config"], function (uid, getMimeType, wcconfig) {
+	"use strict";
+
+	var instance = new FileUtil();
+
+		/**
+		 * Used when checking the newly created file is named with the correct extension.
+		 * If it does not already match any in the array then the extension at index zero will be appended.
+		 **/
+	FileUtil.prototype.mimeToExt = {
```

Comment:
This doesn't (shouldn't?) be public anymore. It was originally public to allow overriding, but this is now achieved via config. Therefore it does not need to be public.

Leaving it public now introduces a new problem, the user may access this property directly, expecting it to reflect the mime map set in the config, but instead they will get unpredictable results:
* If they happen to have previously called `fixFileExtension` they will get the map with config overrides
* If they haven't previously called it they will get the default map

If it MUST be public (it probably doesn't?) then perhaps use an accessor method or similar technique.